com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -783,8 +789,14 @@ half3 VertexLighting(float3 positionWS, half3 normalWS)
 
 #ifdef _ADDITIONAL_LIGHTS_VERTEX
     uint lightsCount = GetAdditionalLightsCount();
+    #ifdef _USE_WEBGL1_LIGHTS
+    for (int lightIndex = 0; lightIndex < _WEBGL1_MAX_LIGHTS; ++lightIndex)
+    {
+        if (lightIndex >= lightsCount) break;
+    #else
     for (uint lightIndex = 0u; lightIndex < lightsCount; ++lightIndex)
     {
+    #endif
```

Comment:
Suggestion, since this code is repeated in a few places and users might want to write custom shaders that work on WebGL 1, maybe we could provide a macro, LIGHT_LOOP_BEGIN(lightsCount) / LIGHT_LOOP_END or something similar?

Reply:
Changed the code to use LIGHT_LOOP_BEGIN/END macros